apps/web-giddh/src/app/settings/integration/setting.integration.component.html

```diff
@@ -394,117 +395,120 @@ <h2 style="float: left; font-size: 20px">ICICI Bank</h2>
+                                                        <p>Unverifed</p>
+                                                        <img src="assets/images/close-round.svg" class="check-search" alt="">
+                                                    </div>
+                                                </fieldset>
+                                            </ng-template>
+                                        </sh-select>
+                                    </div>
+                                    <div class="custom-select select-caret pos-rel" *ngIf="item.otpType==='BANK'">
+                                        <input type="text" class="form-control" disabled placeholder="Registered" name="approvalUniqueName{{index}}" value="">
+                                    </div>
                                 </div>
-                            </div>
-                            <div class="form-group plus-cross-btn col-sm-1 pl-0 pr-0">
-                                <ng-container *ngIf="index===0"> <label class="d-flex">&nbsp;</label></ng-container>
-                                <div class="mt-15">
-                                    <i (click)="deleteAmountRangeRow(item, ind+1)" class="icon-trash cursor-pointer"></i>
+                                <div class="form-group plus-cross-btn col-sm-1 pl-0 pr-0">
+                                    <ng-container *ngIf="index===0"> <label class="d-flex">&nbsp;</label></ng-container>
+                                    <div class="mt-15">
+                                        <i (click)="deleteAmountRangeRow(item, ind+1)" class="icon-trash cursor-pointer"></i>
+                                    </div>
                                 </div>
                             </div>
+                        </ng-container>
+
+                        <div>
+                            <a (click)="addNewAmountRangeRow(ind+1)" class="btn btn-link pl-0">+ Add New Row</a>
                         </div>
-                    </ng-container>
-                    <div>
-                        <a href="javascript:void" (click)="addNewAmountRangeRow(ind+1)" class="btn btn-link pl-0">+ Add New Row</a>
-                    </div>
-                    <!-- New code -->
+                        <!-- New code -->
 
-                    <div class="row">
-                        <div class="form-group">
-                            <label>&nbsp;</label>
-                            <br>
-                            <button type="button" class="btn btn-success mrR1" (click)="updateICICDetails(regAcc, ind)">Update
+                        <div class="row">
+                            <div class="form-group">
+                                <label>&nbsp;</label>
+                                <br>
+                                <button type="button" class="btn btn-success mrR1" [disabled]="!paymentFormUpdate.form.valid" (click)="updateICICDetails(regAcc, ind)">Update
```

Comment:
Incorrect spelling of **updateICICDetails**
Consider using `updateIciciDetails `

Reply:
its existing